openedx/features/course_experience/views/course_home_messages.py

```diff
@@ -124,7 +125,7 @@ def _register_course_home_messages(request, course, user_access, course_start_da
             ),
             title=Text(_('You must be enrolled in the course to see course content.'))
         )
-    if not user_access['is_anonymous'] and not user_access['is_staff'] and not user_access['is_enrolled']:
+    if not user_access['is_anonymous'] and not user_access['is_staff'] and not user_access['is_enrolled'] and not (settings.FEATURES.get('COURSE_DEFAULT_INVITE_ONLY') or course.invitation_only):
```

Comment:
If `course.invitation_only` can we show the same message used on the course_about.html page? 

`<span class="register disabled">${_("Enrollment in this course is by invitation only")}</span>`

Reply:
Updated